ios/PurchasesHybridCommon/PurchasesHybridCommon/CustomerInfo+HybridAdditions.swift

```diff
@@ -1,17 +1,17 @@
 //
-//  PurchaserInfoExtensions.swift
+//  CustomerInfo+HybridAdditions.swift
 //  PurchasesHybridCommon
 //
 //  Created by Andrés Boedo on 4/13/22.
 //  Copyright © 2022 RevenueCat. All rights reserved.
 //
 
 import Foundation
-import Purchases
+import RevenueCat
 
-@objc public extension Purchases.PurchaserInfo {
+@objc public extension CustomerInfo {
 
-    @objc var dictionary: [String: Any] {
+    var dictionary: [String: Any] {
```

Comment:
Well... this doesn't rely on `rawData` mainly because that's a brand new field and this is pretty old logic that I just adapted 😅 
I think we should eventually actually move this logic to the main repo, and maybe even move to use `rawData` instead. `rawData` isn't really gonna change since that would break old SDK versions

Reply:
I was mainly thinking out loud hehe.